packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -447,6 +447,7 @@ class SemanticsData with Diagnosticable {
     required this.maxValueLength,
     required this.currentValueLength,
     required this.headingLevel,
+    this.linkUri,
```

Comment:
do we assert if this is provided then the `link` must be true?

Reply:
I wasn't sure if that would be a good idea or not, so I didn't do it. I thought we may want to allow people to do something like this:
```dart
Semantics(
  link: true,
  child: Semantics(
    linkUri: '...',
  ),
)
```

Reply:
I'm taking over this PR. I don't have a lot of familiarity with semantics and I have the same question as Mouad. Is the snippet he posted something we want to support?

Reply:
how would you handle cases link = false but linkUri is not empty

Reply:
a ping to this in case you missed this.

Reply:
I added asserts for `linkUri != null => link == true`